### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by jackfranklin**
Happy to change `prettier.config.js` to what you like, tried to take it from how the code looked to me :)

### CODE REVIEW

**Comment by jackfranklin** [comment]
Sure thing :)

File: package.json [hunk 1]
```diff
@@ -1,125 +1,257 @@
...
+  "scripts": {
+    "vscode:prepublish": "npm run compile",
+    "clean": "rimraf ./out",
+    "compile": "npm run clean && tsc -p ./",
+    "watch": "npm run clean && tsc -watch -p ./",
+    "postinstall": "node ./node_modules/vscode/bin/install",
+    "test": "npm run compile && node ./node_modules/vscode/bin/test",
+    "sync-changelog": "node ./node_modules/github-releases-renderer/index.js run aioutecism amVim-for-VSCode CHANGELOG.template.md CHANGELOG.md -c 0",
+    "prettier-check": "prettier --list-different 'src/**/*.ts'",
+    "prettier-write": "prettier --write 'src/**/*.ts'"
```
Comment (aioutecism) [inline package.json]: I think we can add `test` folder to the path too.
`'{src,test}/**/*.ts'`

File: package.json [hunk 2]
```diff
@@ -1,125 +1,257 @@
...
+        "command": "workbench.action.closeMessages",
+        "when": "globalMessageVisible"
+      },
+      {
+        "key": "escape",
+        "command": "workbench.action.closeQuickOpen",
+        "when": "inQuickOpen"
+      }
+    ]
+  },
```
Comment (aioutecism) [inline package.json]: The indention of this file changed a lot probably due to the different version of `npm` we are using (Mine is `5.6.0`).
Cloud you please keep the original indention style?

Comment (aioutecism) [reply package.json]: And, `keybindings` block is much more readable when properties are joined into 1 line.

**Comment by aioutecism** [comment]
I added some commit to `master` branch after you created this PR. Cloud you please rebase this branch on top of `master`? Sorry for the extra work...

**Comment by jackfranklin** [comment]
Closing in favour of #216

**Pull request closed without merging**